media/js/firefox/whatsnew_38/whatsnew-video.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function(){
```

Comment:
No big gain, but one less event handler and keeping consistent with other JS files.